R/data-initial-placeholder.R

```diff
@@ -0,0 +1,11 @@
+
+#' Initial Placeholder Data
+#'
+#' The initial data.frame is generated as a placeholder until a more streamlined
+#' process is implemented. It contains the initial run where all of CRAN was
+#' scored using riskmetric v0.2.1 on 2023-06-21. It contains 19,715 observations
+#' (one row per package) and 24 variables, 18 of which are riskmetric
+#' assessments.
```

Comment:
The data format description is best captured in a `@format` tag

```suggestion
#' The initial data.frame is generated as a placeholder until a more streamlined
#' process is implemented. It contains the initial run where all of CRAN was
#' scored using riskmetric v0.2.1 on 2023-06-21.
#'
#' @format A `data.frame` containing 19,715 observations
#'   (one row per package) and 24 variables, 18 of which are riskmetric
#'   assessments.
#'
```